swabbie-core/src/main/kotlin/com/netflix/spinnaker/swabbie/model/OnDemandMarkData.kt

```diff
@@ -0,0 +1,16 @@
+package com.netflix.spinnaker.swabbie.model
+
+import com.netflix.spinnaker.swabbie.notifications.Notifier
+
+data class OnDemandMarkData(
+  var projectedSoftDeletionStamp: Long,
+  var projectedDeletionStamp: Long,
+  var markTs: Long? = null,
+  var resourceOwner: String = "swabbie@spinnaker.io",
+  var notificationInfo: NotificationInfo? = NotificationInfo(
+    recipient = resourceOwner,
+    notificationType = Notifier.NotificationType.EMAIL.name,
+    notificationStamp = 1538589958608,
```

Comment:
Is a static timestamp desired?

Reply:
I'm not sure? This is for testing, so providing these default values let you submit a marked resource w/o specifying all notification info lets the resource actually be cleaned (resource won't be cleaned if it hasn't been notified first). The time doesn't particularly matter. You can override it in your payload body when you submit via the api. I'd say static is fine for now.

Reply:
Even if the intention today is for testing-only purposes, we can't be sure that its scope will remain a testing-only function in the future. I'd say it shouldn't have any default value assigned and should be explicitly set by the caller. These sort of assumptions have a higher chance of creating bugs some years down the line.

Reply:
Great point. Removed the timestamp but kept notification for easy testing.